apps/minifront/src/components/swap/swap-form/token-swap-input.tsx

```diff
@@ -85,7 +85,7 @@ export const TokenSwapInput = () => {
     } else {
       return priceHistory.load();
     }
-  }, [assetIn, assetOut]);
+  }, [assetIn, assetOut, priceHistory]);
```

Comment:
```suggestion
  }, [assetIn, assetOut, priceHistory.load]);
```

(Otherwise, this effect will run any time _any_ property of the `priceHistory` slice changes.)

Reply:
ah, this might have been related to some of the state churn i mentioned. thanks!